src/markdown/handbook/tools/jira/issues/communication-design-request.md

```diff
@@ -11,7 +11,13 @@ Request related to the creation of an asset handled by the Communication Team.
 
 ### When to create a Communication Request?
 
-- TO DO
+Consider creating a ticket whenever there's a need for visual design material, related to internal and external communication, HR campaigns, Marketing collateral or any kind of digital or print asset or media request. More information can be found on [Confluence](https://liferay.atlassian.net/wiki/spaces/DSGN/pages/1806699293/Creating+a+Design+Request+Ticket).
+
+### Comm Design Jira Ticket Sizing
+
+TSize is a way to generally categorize the amount of effort/time a ticket may take to fulfill. The TSize will be designated by the ticket delegator according to his/her’s estimation of the project and the skills of the assigned designer. 
+
+There is more information on this [document](https://docs.google.com/document/d/1Se6FNe8UDpyjfWac1FgUD17BMKO7rEcYoEgjlEK5jrc/edit) over the amount of work of a communication ticket.
```

Comment:
any reason to not move this content out of a Gdoc and into our documentation?

Reply:
I really don't know if this content is relevant or if the comm team will keep using this concept. I going to check with them and we can update it later if this make sense.